integration-tests/pkgJson-restore.spec.js

```diff
@@ -516,7 +514,7 @@ describe('update pkg.json to include platforms in config.xml', function () {
         expect(pkgJson.dependencies[browserPlatform]).toBeUndefined();
         expect(pkgJson.dependencies[androidPlatform]).toBeUndefined();
         emptyPlatformList().then(function () {
-            return prepare();
+            return prepare({'fetch': true});
```

Comment:
@stevengill In this file, I've added the fetch flag to most/all of the tests to fix them. This did not work the same way for test 005 and 007. You do need to add in **fetch** here (just like all of the other tests) or else you will get  this failure -->`Expected No platforms added to this project. Please use `cordova platform add <platform>`. to be undefined.`

However, adding the fetch flag here, will lead to other 005 and 007 tests failures as shome below. (FYI they are passing independently.) -->

```cp: no such file or directory: /private/var/folders/13/dkz30nws1dv3dzlh6dvthwzm0000gn/T/e2e-test/platform_test_pkgjson/project/node_modules/cordova-android/bin/templates/project/wrapper.gradle```